AmazonChimeSDKDemo/AmazonChimeSDKDemo/MeetingModel.swift

```diff
@@ -185,6 +185,30 @@ class MeetingModel: NSObject {
     func isVoiceFocusEnabled() -> Bool {
         return currentMeetingSession.audioVideo.realtimeIsVoiceFocusEnabled()
     }
+    
+    func setLiveTranscriptionEnabled(enabled: Bool) {
+        let action = enabled ? "enable" : "disable"
+        startOrStopTranscription(action: action)
```

Comment:
little confused any reason why you would change boolean to string? It doesn't seems to be used either other than comparing.

Reply:
removed, directly checking with bool now.